conan/cli/commands/lock.py

```diff
@@ -59,7 +60,7 @@ def lock_create(conan_api, parser, subparser, *args):
                                                   clean=args.lockfile_clean)
     conanfile_path = os.path.dirname(graph.root.path) \
         if graph.root.path and args.lockfile_out is None else cwd
-    conan_api.lockfile.save_lockfile(lockfile, args.lockfile_out or "conan.lock", conanfile_path)
+    conan_api.lockfile.save_lockfile(lockfile, args.lockfile_out or LOCKFILE, conanfile_path)
```

Comment:
```suggestion
    conan_api.lockfile.save_lockfile(lockfile, args.lockfile_out or "conan.lock", conanfile_path)
```
Better leave this as-is.

Avoids the ``LOCKFILE`` import from ``from conan.internal``. My other PR will remove this import.